extensions/braze/actions/sendSms/config/fields.ts

```diff
@@ -0,0 +1,32 @@
+import { type Field, FieldType } from '@awell-health/extensions-core'
+import { type ZodTypeAny, z } from 'zod'
+
+export const fields = {
```

Comment:
Similar Q: how's the recipient defined?

Reply:
same answer as for email, I'll ask John once he gets in touch about api key